aten/src/ATen/native/mkl/SparseBlasImpl.cpp

```diff
@@ -367,6 +367,15 @@ void addmm_out_sparse_csr(
         alpha,
```

Comment:
Judging from the lines above `addmm_dense_result` should also work with `kSparseBsr`.

Reply:
I wonder, is it that hard to add these transposition flags?

Reply:
Probably best left to a follow up.